apps/learner/src/lib/components/FloatingPlayer.svelte

```diff
@@ -31,27 +31,25 @@
   };
 </script>
 
-<div class="z-100 fixed bottom-6 left-6 right-6">
-  <a
-    href={to}
-    class="inset-shadow-sm flex items-center gap-x-3 rounded-full bg-white/90 p-3 shadow-lg backdrop-blur-sm"
-  >
-    <!-- Temporary album placeholder -->
-    <div class="h-12 w-12 rounded-full bg-black"></div>
+<a
+  href={to}
+  class="inset-shadow-sm flex items-center gap-x-3 rounded-full bg-white/90 p-3 shadow-lg backdrop-blur-sm"
+>
+  <!-- Temporary album placeholder -->
+  <div class="h-12 w-12 rounded-full bg-black"></div>
 
-    <div class="flex-1 truncate text-sm font-medium">
-      {title}
-    </div>
+  <div class="flex-1 truncate text-sm font-medium">
+    {title}
+  </div>
 
-    <button
-      class="flex cursor-pointer items-center rounded-full px-4 py-2 hover:bg-slate-50"
-      onclick={handlePlay}
-    >
-      {#if isplaying}
-        <Pause />
-      {:else}
-        <Play />
-      {/if}
-    </button>
-  </a>
-</div>
+  <button
+    class="flex cursor-pointer items-center rounded-full px-4 py-2 hover:bg-slate-50"
+    onclick={handlePlay}
+  >
+    {#if isplaying}
+      <Pause />
+    {:else}
+      <Play />
+    {/if}
+  </button>
```

Comment:
I'm gonna extract this as a component because of the centralized state sync when a podcast is played for Learning Unit, Floating Player, and the Full Player Modal but should not be a blocker for this MR